systemtests/bin/symbolicate.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+# Sends a stack for symbolication with a Symbols server using the
+# Symbolicate API.
+
+# Usage: ./bin/symbolicate.py FILE
+
+import json
+import os
+import sys
+
+import click
+import jsonschema
+import requests
+
+
+def load_schema(path):
+    with open(path, "r") as fp:
+        schema = json.load(fp)
+    jsonschema.Draft7Validator.check_schema(schema)
+    return schema
+
+
+@click.command()
+@click.option(
+    "--api-url",
+    default="https://symbols.mozilla.org/symbolicate/v4",
+    help="The API url to use.",
+)
+@click.option(
+    "--api-version",
+    default=4,
+    type=int,
+    help="The API version to use; 4 or 5; defaults to 4.",
+)
+@click.option(
+    "--debug/--no-debug", default=False, help="Whether to include debug info."
+)
+@click.option("--verify/--no-verify", default=False, help="Whether to verify response.")
+@click.argument("stackfile", required=False)
+@click.pass_context
+def symbolicate_stack(ctx, api_url, api_version, debug, verify, stackfile):
+    if not stackfile and not sys.stdin.isatty():
+        data = click.get_text_stream("stdin").read()
+
+    else:
+        if not os.path.exists(stackfile):
+            raise click.BadParameter(
+                "Stack file does not exist.",
+                ctx=ctx,
+                param="stackfile",
+                param_hint="stackfile",
+            )
+
+        with open(stackfile, "r") as fp:
+            data = fp.read()
+
+    if api_version not in [4, 5]:
+        raise click.BadParameter(
+            "Not a valid API version number. Must be 4 or 5.",
+            ctx=ctx,
+            param="api_version",
+            param_hint="api_version",
+        )
+
+    if verify:
```

Comment:
`verify` is actually about two different modes of running. I think I should split this out into two separate commands like I did with `make-stacks`.